ludwig/distributed/base.py

```diff
@@ -20,11 +27,27 @@ class DistributedStrategy(ABC):
     """
 
     @abstractmethod
-    def wrap_model(self, model: nn.Module) -> nn.Module:
+    def prepare(
+        self,
+        model: nn.Module,
+        trainer_config: "ECDTrainerConfig",
+        base_learning_rate: float,
+    ) -> Tuple[nn.Module, Optimizer]:
```

Comment:
It might be useful to add a docstring here indicating what the `prepare` function is supposed to do. From below, it seems like we're returning the model/wrapping the model with the relevant strategy initialization + creating the optimizer?